### ISSUE TITLE
"erro no retorno" ao pressionar pra gravar

### ISSUE THREAD

**Issue by amimaro** [patrickmonteiro/quasar-speech-api#15]
Quando pressiono pra gravar, aparece a overlay de carregamento mas some logo em seguida.

No console, aparece a mensagem "erro no retorno".

Se o Contínuo tiver ligado, acontece a mesma coisa só que sem a mensagem "erro no retorno" e o botão de stop fica na página.

Tem limitação de browser esse app?

**Comment by patrickmonteiro** [comment]
@amimaro sim, algumas dessas WEB API's são experimentais, e só existem no chrome ou navegadores baseados no chromium.
Atualmente funciona bem no chrome desktop e mobile, no safari parcialmente.

**Comment by patrickmonteiro** [comment]
@amimaro Um PR com detecção de browser seria bem vindo e bem simples de fazer já que o Quasar possui a detecção de plataforma
